indra/explanation/reporting.py

```diff
@@ -173,7 +173,7 @@ def stmts_from_pybel_path(path, model, from_db=True, stmts=None):
         hashes = set()
         for j in range(len(edges)):
             try:
-                hashes.add(edges[j]['annotations']['stmt_hash'])
+                hashes.add(list(edges[j]['annotations']['stmt_hash'])[0])
```

Comment:
We're assuming that `edges[j]['annotations']['stmt_hash']` is of type `Mapping[str,bool]` and even further that it has exactly one key. This change extracts that key